test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Xml.Tests.ps1

```diff
@@ -67,6 +67,43 @@
 		$cmd = [Microsoft.PowerShell.Commands.ConvertToXmlCommand]::new()
 		$cmd.NoTypeInformation = $true
 		$cmd.NoTypeInformation | Should Be $true
-	}
+    }
+
+    It "Serialize primitive type" {
+        [int] $i = 1
+        $x = $i | ConvertTo-Xml
+        $x.Objects.Object.Type | Should BeExactly $i.GetType().ToString()
+        $x.Objects.Object."#text" | Should BeExactly $i
+    }
+
+    It "Serialize dictionary type" {
```

Comment:
If you're using terminology from the source code, then I would call this test
"Serialize ContainerType.Dictionary type"